newrelic/core/agent_streaming.py

```diff
@@ -44,7 +44,7 @@ class StreamingRpc(object):
     )
     OPTIONS = [("grpc.enable_retries", 0)]
 
-    def __init__(self, endpoint, stream_buffer, metadata, record_metric, ssl=True):
+    def __init__(self, endpoint, stream_buffer, metadata, record_metric, ssl=True, compression=True):
```

Comment:
Let's default the StreamingRPC to False, let the agent settings handle the default value for actual applications. This way testing behavior doesn't change when manually creating and testing the StreamingRPC.

Reply:
I can do that--Any thoughts on setting it to `None` instead and then using `global_settings_dump()` to set the value if nothing has been passed in?